networking/src/Parsers/TeamAnswerParser.ts

```diff
@@ -0,0 +1,86 @@
+import { isNullOrUndefined } from "../IApiClient";
```

Comment:
With removing `APIClient`, there'll be no usage for `IApiClient` as well, might worth to move this method to a global file. I believe we can create `global.ts` to be publicly available.